nextjs-app/app/dev-only/tailwind-demo/page.tsx

```diff
@@ -0,0 +1,185 @@
+export default async function HomePage() {
```

Comment:
Can we change this component name to `TailwindDemo` or something make more sense ?

Reply:
Sure, my bad
[c2c5511](https://github.com/thementorship-tw/thementorship.tw/pull/21/commits/c2c55111159d39455b162046ca054fa1ce551b0f)